portability-core/src/main/java/org/dataportabilityproject/job/JobDao.java

```diff
@@ -39,7 +39,7 @@
    * The value ASSIGNED_WITHOUT_AUTH_DATA indicates the client has submitted all data required, such as the
    * encrypted auth data, in order to begin processing the portability job.
    */
-  enum JobState {
+  public enum JobState {
```

Comment:
This also looks like (outside of this class) its only used in Test, is there a way we can limit the visibility?

Reply:
Annotated with @VisibleForTesting